
Why do I have to fix those bugs. Honest! Does noone else get unresolved
symbols for any other Gtk+ and Gnome related crap^Wpackage?

Are those Gnome folks _that_ dump to get their own crap^Wfluff^Wdung building?

  - Rene Rebe <rene@exactcode.de>

--- pango-1.6.0/pangoxft.pc.in.orig	2004-09-21 01:51:34.236980753 +0200
+++ pango-1.6.0/pangoxft.pc.in	2004-09-21 01:51:49.255521901 +0200
@@ -6,6 +6,6 @@
 Name: Pango Xft
 Description: Xft font support for Pango
 Version: @VERSION@
-Requires: pango
+Requires: pango pangoft2
 Libs: -L${libdir} -lpangoxft-@PANGO_API_VERSION@ @PKGCONFIG_XFT_LIBS@
 Cflags: -I${includedir}/pango-1.0 @XFT_CFLAGS@
